packages/nodes-base/nodes/Form/templates.ts

```diff
@@ -403,18 +403,51 @@ export const createPage = (
 	instanceId?: string,
 ) => {
 	const { formHtml, variables, validationCases } = prepareFormGroups(formFields);
+	const head = `
```

Comment:
could have used handlebars for these templates as well?

Reply:
I like idea, need to check how feasible it is

Reply:
@mutdmour 
updated, page would be rendered with handlebars now